### PR TITLE
[MRG+1] Make scrapy available in shell without explicit import statement

### PR DESCRIPTION
**Pull request by redapple**
Maybe it's just me but I'm always a bit annoyed to have to do `import scrapy` in the shell.
Thoughts?

### CODE REVIEW

**Comment by eliasdorneles** [comment]
Just tested here, MUCH better! :)

File: scrapy/shell.py [hunk 1]
```diff
@@ -115,6 +115,9 @@ def fetch(self, request_or_url, spider=None):
         self.populate_vars(response, request, spider)
 
     def populate_vars(self, response=None, request=None, spider=None):
+        import scrapy
+
+        self.vars['scrapy'] = scrapy
```
Comment (kmike) [inline scrapy/shell.py]: I think it worths mentioning in get_help, something like "scrapy is already imported"

Comment (eliasdorneles) [inline scrapy/shell.py]: +1.

My suggestion:

```
        b.append("Available Scrapy objects:")
        b.append('  scrapy     scrapy module (contains scrapy.Request, scrapy.Selector, etc)')
        ...
```

Comment (redapple) [inline scrapy/shell.py]: +1

**Pull request merged**